### ISSUE TITLE
How to work with "external_id"

### ISSUE THREAD

**Issue by rretanubun** [thingsboard/thingsboard#11790]
**Component**
 * UI

**Description**
I am trying to do Version Control and want to explore using [ExternalId](https://thingsboard.io/docs/user-guide/version-control/#entity-external-id) I can't seem to export  `externalID` (or `external_id`) (either via the GUI or via the python tb-rest-client)

**Environment**

 * OS:  Ubuntu 22.04.5 LTS
 * ThingsBoard: 3.7.0 inside docker with `tb-rest-client` version 3.7.0
 * Browser: Chrome Version 129.0.6668.60 (Official Build) (64-bit)

**What I have done so far**
* I am aware of [issue-9857](https://github.com/thingsboard/thingsboard/pull/9857)
* I am using the example `thermostat rule chain` if it helps, but my goal is to export import various entities using external id
* When I export using `thingsboard GUI (as a `tenant administator` user) the exported JSON does not have "external_id"
* When I export using `python tb-rest-client` I got `"external_id": null` entries in the export
* I then take the `python tb-rest-client` exported content and then manually inject `external_id` like this:
```
{
  "ruleChain": {
    "name": "ThermostatWithExtID", <== Name changed from default
    "type": "CORE",
    "firstRuleNodeId": null,
    "root": false,
    "debugMode": false,
    "configuration": null,
    "additionalInfo": {
      "description": ""
    }
  },
  "metadata": {
    "firstNodeIndex": 6,
    "nodes": [
      {
        "type": "org.thingsboard.rule.engine.telemetry.TbMsgTimeseriesNode",
        "name": "Save Timeseries",
        "debugMode": false,
        "external_id": "92ee852a-d1ef-454c-b152-165b35c759ca", <<-- manually typed
<snip>
```
* I am able to import the "ThermostatWithExtID" rule chain, but when I re-export it from GUI or from `tb-rest-client` I am not able to get the `external_id` field.

What am I missing?

Thanks for everyone's time!

**Comment by mykhailokornieiev** [comment]
Hi

From [documentation](https://thingsboard.io/docs/user-guide/version-control/#entity-external-id):

> The field is used to identify the same Entity in case of import and export between multiple environments.

For example, External ID is created after you have created a device on a so-called prod server, created a version of this device, then imported this device to the so-called test server. Then you make changes to the device on one of these servers, import these changes to the other server. And only after that, an external ID is created for further synchronization between these same entities.

<img width="1024" height="662" alt="Image" src="https://github.com/user-attachments/assets/3078b5db-dbc3-473f-98a9-69d205850853" />

As the community has not provided assistance on this matter over an extended period, I am closing this ticket due to inactivity. If it is still relevant, feel free to open a new ticket with a full description of your request.
